### DISCUSSION TITLE
FAQ schema for new accordion section

### DISCUSSION THREAD

**Discussion by fentisam** [elementor/elementor#24143] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

FAQ schema when using the new accordion section

### Describe the solution you'd like

The ability to switch between the new and old accordion widgets and allow the FAQ schema to be active.

**Comment by fabigrafo** [comment]
In the new 3.17 version of free and pro there is it:

### Accordion Improvement - Make Your FAQ Frequently Found 

The Accordion Widget, with its nested capabilities, allows you to place Widgets inside Accordion items, giving you unlimited layout options. In version 3.17, the Accordion Widget also improves the SEO compatibility of your content by adding a FAQ Schema control.

The FAQ Schema adds a specialized markup to your page’s code that contains a list of questions and answers. Search engines can read this markup and use it to generate rich code.
